vermouth/tests/test_hypothesis_graph_utils.py

```diff
@@ -0,0 +1,92 @@
+# Copyright 2018 University of Groningen
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.import pytest
+
+import networkx as nx
+
+import hypothesis.strategies as st
+from hypothesis import given, note, settings
+
+from hypothesis_networkx import graph_builder
+import vermouth
+
+
+max_nodes = 10
+attrnames = ['attr1', 'attr2']
+node_data = st.dictionaries(keys=st.sampled_from(attrnames), values=st.integers(min_value=0, max_value=max_nodes))
+
+attrs = st.lists(st.sampled_from(attrnames), unique=True, min_size=0, max_size=2)
+
+MCS_builder = graph_builder(node_data=node_data, min_nodes=0, max_nodes=max_nodes, node_keys=st.integers(max_value=max_nodes, min_value=0))
+
+
+@settings(max_examples=500)
+@given(graph1=MCS_builder, graph2=MCS_builder, attrs=attrs)
+def test_maximum_common_subgraph(graph1, graph2, attrs):
+    expected = vermouth.graph_utils.categorical_maximum_common_subgraph(graph1, graph2, attrs)
+    
+    found = vermouth.graph_utils.maximum_common_subgraph(graph1, graph2, attrs)
+    
+    note(attrs)
```

Comment:
It would be better with something to tell what the note is about. Same as for the prints above.